src/carousel/carousel.scss

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../text/index';
+@import '../button/button';
+
+@mixin iui-carousel {
+  display: flex;
+  flex-direction: column-reverse;
+  overflow: hidden;
+}
+
+@mixin iui-carousel-slider {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  @media (prefers-reduced-motion: no-preference) {
+    transition: transform $iui-speed-slow ease-in-out;
+  }
+
+  &-item {
+    width: 100%;
+    flex-shrink: 0;
+    box-sizing: border-box;
+  }
+}
+
+@mixin iui-carousel-navigation {
```

Comment:
I think this whole navigation needs some focus styling (and tabindex).

I imagine a keyboard user will want to tab into the carousel in the following sequence:
1. get to the play/pause button first so that autoplay can be paused (using enter key)
2. tab to the slide control and press left/right arrow keys to change slides. They will _not_ want to focus the actual dots or the prev/next buttons if they can just use arrow keys.
3. tab to the interactable content in the current slide.

Reply:
I dont know.. I would say, if I tab into Carousel, Enter/Space should trigger play/pause. Then left/right arrows should switch slides. And then additional tab to get into carousel content. That intuitive to me, but need to read w3 recommendation on this one, if they have it :/

Reply:
W3C does not even recommend using prev/next buttons together with dots, so our situation is a bit weird. But they do recommend the arrow keys https://www.w3.org/TR/wai-aria-practices-1.2/examples/carousel/carousel-2-tablist.html

Reply:
ahh.. getting weird..
But current solution, when focus is on play button and pressing tab moves you to the prev arrow feels wrong.

Reply:
Maybe we put `tabindex="-1"` on the left / right buttons since the dots are going to use the left / right arrow keys similar to paginator.  At that point, the dots & left / right buttons would be doing the same thing pretty much, right?

Reply:
One of the options, I guess.. Just paginator left/right buttons are also focusable by tabbing into them..

Reply:
> I would say, if I tab into Carousel, Enter/Space should trigger play/pause. Then left/right arrows should switch slides. And then additional tab to get into carousel content.

I implemented this approach so the user can now focus the whole carousel instead of having to tab into those individual pieces (which all have tabindex=-1 now). See [`0be5873`](https://github.com/iTwin/iTwinUI/pull/377/commits/0be587392c44815404a5233d50743a5b94733f11)

Reply:
@FlyersPh9 I think we could change the ordering back to normal now?

Reply:
HTML structure is back to the way it was previously.